docs/source/apps/projinfo.rst

```diff
@@ -0,0 +1,312 @@
+    |    [--pivot-crs none|{auth:code[,auth:code]*}]
+    |    [--boundcrs-to-wgs84]
+    |    [--main-db-path path] [--aux-db-path path]*
+    |    [--identify]
+    |    [--c-ify] [--single-line]
+    |    {object_definition} | (-s {srs_def} -t {srs_def})
+
+    where {object_definition} or {object_definition} is a PROJ string, a
+    WKT string, an object name or a AUTHORITY:CODE
+    (where AUTHORITY is the name of a CRS authority and CODE the code of a CRS
+    found in the proj.db database).
+
+Description
+***********
+
+:program:`projinfo` is a program that can query information on a geodetic object,
+coordinate reference system (CRS) or coordinate operation, when the -s and -t
+options are specified, and display it under different formats (PROJ string, WKT string).
+
+It can also be used to query coordinate operations available between two CRS.
+
+The program is named with some reference to the GDAL `gdalsrsinfo` that offers
+partly similar services.
+
+
+The following control parameters can appear in any order:
+
+.. program:: projinfo
+
+.. option:: -o formats
+
+    formats is a comma separated combination of:
+    `all`,`default`,`PROJ`,`WKT_ALL`,`WKT2_2015`,`WKT2_2018`,`WKT1_GDAL`,`WKT1_ESRI`.
+
+    Except 'all' and 'default', other formats can be preceded by `-` to disable them".
+
+.. option:: -k crs|operation
+
+    When used to query a single object with a AUTHORITY:CODE, determines the (k)ind of the object
+    in case there are CRS or coordinate operations with the same CODE.
+    The default is crs.
+
+.. option:: --sumary
+
+    (only used for coordinate operation computation)
+
+    When listing coordinate operations available between 2 CRS, return the
+    result in a summary format, mentionning only the name of the coordinate
+    operation, its accuracy and its area of use.
+
+.. option:: -q
+
+    Turn on quiet mode. Quiet mode is only available for queries on single objects,
+    and only one output format is selected. In that mode, only the PROJ or WKT
+    string is displayed, without other introduction output. The output is then
+    potentially compatible of being piped in other utilities.
+
+.. option:: --area name_or_code
+
+    (only used for coordinate operation computation)
+
+    Specify an area of interest to restrict the results when researching
+    coordinate operations between 2 CRS. The area of interest can be specified either
+    as a name (e.g "Denmark - onshore") or a AUTHORITY:CODE (EPSG:3237)
+    This option is exclusive of --bbox.
+
+.. option:: --bbox west_long,south_lat,east_long,north_lat
+
+    (only used for coordinate operation computation)
+
+    Specify an area of interest to restrict the results when researching
+    coordinate operations between 2 CRS. The area of interest is specified as a
+    bounding box with geographic coordinates, expressed in degrees in a
+    unspecified geographic CRS.
+    west_long and east_long should be in the [-180,180] range, and
```

Comment:
If you write the limits in code notation they stand out a bit more and it is made clear that they are inputs to the bbox: ``` `west_long` ```, ``` `east_long` ```, ``` `south_lat` ``` and ``` `north_lat` ```